vmm/src/vm.rs

```diff
@@ -267,12 +281,17 @@ impl Vm {
             return Err(Error::CapabilityMissing(Cap::SignalMsi));
         }
 
-        if !kvm.check_extension(Cap::TscDeadlineTimer) {
-            return Err(Error::CapabilityMissing(Cap::TscDeadlineTimer));
+        #[cfg(target_arch = "x86_64")]
```

Comment:
Refactor out into a function in the arch crate `check_required_kvm_extensions()` ?